lib/class-results-list-table.php

```diff
@@ -546,9 +546,19 @@ public function single_row( $post ) {
 		$post = get_post( $post );
 
 		$GLOBALS['post'] = $post; // WPCS: override ok.
-
 		setup_postdata( $post );
-		parent::single_row( $post );
+
+		$classes = '';
```

Comment:
You should make this an array. Core does a little less work when it's an array vs when it's a string:

```php
function get_post_class( $class = '', $post_id = null ) {
	...
	$classes = array();

	if ( $class ) {
		if ( ! is_array( $class ) ) {
			$class = preg_split( '#\s+#', $class );
		}
		$classes = array_map( 'esc_attr', $class );
	} ...
```